packages/studioCMS/package.json

```diff
@@ -67,18 +65,22 @@
     "mrmime": "^2.0.0",
     "oslo": "^1.2.1",
     "shiki": "^1.9.1",
-    "unpic": "^3.18.0",
-    "unocss": "^0.61.0"
+    "unocss": "^0.61.0",
+    "unpic": "^3.18.0"
   },
   "peerDependencies": {
     "@astrojs/db": ">=0.11.6",
-    "astro": ">=4.10.3"
+    "@types/react": "^18.3.3",
+    "astro": ">=4.10.3",
+    "react": "^18.3.1"
```

Comment:
yeah, make sense, both react and @types/react?

Reply:
I think just react needs to be moved to an Optional state... but @jdtjenkins could probably give a better answer since i know he had to deal with this when he made the devtoolbar app for AIK

Reply:
> I think just react needs to be moved to an Optional state... but @jdtjenkins could probably give a better answer since i know he had to deal with this when he made the devtoolbar app for AIK

yes, so we wait for his answer

Reply:
Yeah for AIK we had `react` , `@types/react` and `react-dom` as optional peer deps if I remember correctly!

```json
{
  "peerDependencies": {
    "react": "^",
    "@types/react": "^",
    "react-dom": "^",
  },
  "peerDependenciesMeta": {
    "react": {
      "optional": true
    },
    "@types/react": {
      "optional": true
    },
    "react-dom": {
      "optional": true
    }
  }
}
```

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta

Reply:
> Yeah for AIK we had `react` , `@types/react` and `react-dom` as optional peer deps if I remember correctly!
> 
> ```json
> {
>   "peerDependencies": {
>     "react": "^",
>     "@types/react": "^",
>     "react-dom": "^",
>   },
>   "peerDependenciesMeta": {
>     "react": {
>       "optional": true
>     },
>     "@types/react": {
>       "optional": true
>     },
>     "react-dom": {
>       "optional": true
>     }
>   }
> }
> ```
> 
> https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta

@jdtjenkins yeah, thank you!